src/visitors/verbatim_var_rename_visitor.cpp

```diff
@@ -60,7 +60,7 @@ std::string VerbatimVarRenameVisitor::rename_variable(std::string name) {
         if (symbol != nullptr) {
             return new_name;
         }
-        std::cerr << "Warning : could not find " << name << " definition in nmodl" << std::endl;
+        std::cerr << "Warning : could not find " << name << " definition in nmodl\n";
```

Comment:
Since you're fixing this, I think this should be rewritten using spdlog (like elsewhere in the code):

```suggestion
        logger.warn("could not find {} definition in nmodl"_fmt(name));
```

and you'll need to get the logger instance from somewhere...

Reply:
If you don't mind, I prefer to delegate the build of the string to spdlog. If log level is ERROR, it will save the string construction. See 0264924

Reply:
You are right of course, I had forgotten how to do the string building correctly.